Update readme documentation (#307)
aerospike/aerospike-backup-service: README.md

The credential object has a new optional `secret-agent` property that points to a secret agent, one of those listed in
the `secret-agents` configuration parameter.
Secret agent is responsible for storing secrets like passwords and TLS certificates.
`password` field can contain either password itself (not recommended), or a path pointing to the
password in the Aerospike Secret Agent (starts with `secrets:`).

Fields `password` and `password-path` are mutually exclusive.

```yaml

>      dto.Credentials:

Reviewer: Remove the left padding from the first line and format.

           enum:
             - INTERNAL
             - EXTERNAL
             - PKI
           type: string
         password:
           description: |-
             The password for the cluster authentication.
             It can be either plain text or path into the secret agent.
           example: testPswd
           type: string
         password-path:
           description: The file path with the password string.
           example: /path/to/pass.txt
           type: string
         secret-agent:
           allOf:
             - $ref: '#/components/schemas/dto.SecretAgent'
           description: |-
             Secret Agent configuration (optional).
             Mutually exclusive with secret-agent-name.
           type: object
         secret-agent-name:
           description: |-
             Secret Agent configuration (optional). Link to one of preconfigured agents.
             Mutually exclusive with secret-agent.
           type: string
         user:
           description: The username for the cluster authentication.
           example: testUser
           type: string
       type: object
```

### Cancel Restore Job

New endpoint:  

> [

Reviewer: ```suggestion
- [
```

> `POST {{baseUrl}}/v1/restore/cancel/:<jobId>`](https://aerospike.github.io/aerospike-backup-service/#/Restore/cancelRestore)

Reviewer: Is a new line here because of auto-formatting?

Author: yes.

Cancel the restore job identified by `<jobId>`. Data that has already been restored will remain intact.

### Cancel Backup Job

New endpoint:  

> [

Reviewer: ```suggestion
- [
```

The credential object has a new optional `secret-agent` property that points to a secret agent, one of those listed in
the `secret-agents` configuration parameter.
Secret agent is responsible for storing secrets like passwords and TLS certificates.

> `password` field can contain either password itself (not recommended), or a path pointing to the
> password in the Aerospike Secret Agent (starts with `secrets:`).
> 
> Fields `password` and `password-path` are mutually exclusive.

Reviewer 2: ```suggestion
`The password` field can contain either the password itself (recommended only for testing), or a path pointing to the password in the Aerospike Secret Agent (starts with `secrets:`).

The fields `password` and `password-path` are mutually exclusive.
```
